src/framework/application.js

```diff
@@ -2318,6 +2313,15 @@ var makeTick = function (_app) {
         application.fire("framerender");
 
         if (application.autoRender || application.renderNextFrame) {
+            // In AUTO mode the resolution is changed to match the canvas size
+            if (application._resolutionMode === RESOLUTION_AUTO) {
+                // Check if the canvas DOM has changed size
+                const canvas = application.graphicsDevice.canvas;
+                if (canvas.width !== canvas.clientWidth || canvas.height !== canvas.clientHeight) {
```

Comment:
So you're comparing canvas resolution (which accounts for devicePixelRatio) against the client width and height (which doesn't). Won't this be different every frame if devicePixelRatio is not 1?

Reply:
Good question, I will take a look

Reply:
Good spot, fixed in a later commit